snap/snapcraft.yaml

```diff
@@ -349,6 +349,15 @@ apps:
     install-mode: disable
     start-timeout: 1m
     plugs: [network]
+  # this service pushes common configuration source into Configuration Provider
+  core-common-config-bootstrapper:
+    after:
+      - security-consul-bootstrapper
+    command: bin/core-common-config-bootstrapper --configDir $SNAP_DATA/config/core-command/res --configProvider --registry 
```

Comment:
need to add `-cf configuration.yaml`

Reply:
Is `--configFile configurational.yaml` equal to `--configDir ./res`?

Reply:
@MonicaisHer The former is to specify a custom configuration file name, while the latter is to set the path to the directory containing that file.

I believe `--configFile configurational.yaml` is needed because the default value is `configuration.toml`.

```
$ ./cmd/core-common-config-bootstrapper/core-common-config-bootstrapper -h
Usage: ./cmd/core-common-config-bootstrapper/core-common-config-bootstrapper [options]
Server Options:
    -cp, --configProvider           Indicates to use Configuration Provider service at specified URL.
                                    URL Format: {type}.{protocol}://{host}:{port} ex: consul.http://localhost:8500
    -o, --overwrite                 Overwrite configuration in provider with local configuration
                                    *** Use with cation *** Use will clobber existing settings in provider,
                                    problematic if those settings were edited by hand intentionally
    -cf, --configFile <name>        Indicates name of the local configuration file. Defaults to configuration.toml
    -p, --profile <name>            Indicate configuration profile other than default
    -cd, --configDir                Specify local configuration directory
    -r, --registry                  Indicates service should use Registry.

Common Options:
        -h, --help                      Show this message
```

Reply:
Correct. This is need until the default is `configuration.yaml` when we switch from TOML to YAML for all configuration files.

Reply:
Also, it is `configuration.yaml`, not `configurational.yaml` ;-)